### PR TITLE
Docs: added TfN logo to sphinx docs

### PR DESCRIPTION
**Pull request by MattBuckley-TfN**
# Changes

Removed logo from repo and added links to TfN white and standard logos in sphinx docs. Linked to TfN banner in the READMEs.
Removed the CAF handbook links and replaced with links to org GitHub page.

# Tasks

- [x] Have unittests been added (testing individual functions and their edge cases) - NA
- [x] Has documentation (including user guide) been updated - Yes
- [x] Have new dependencies been added (or changed) in relevant `requirements.txt` - NA
- [x] Code linting, tests and other workflows pass

### CODE REVIEW

File: {{ cookiecutter.project_slug }}/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-<div align="center" style="background-color: white;">
+<div align="center">
 <a href="https://www.transportforthenorth.com/">
-<img src="https://www.transportforthenorth.com/logo.svg" alt="Transport for the North logo">
+<img
+    src="https://raw.githubusercontent.com/Transport-for-the-North/.github/refs/heads/main/profile/tfn-banner.svg"
+    alt="Transport for the North logo">
```
[the remark this answers is not in the document]
Comment (MattBuckley-TfN) [reply {{ cookiecutter.project_slug }}/README.md:5]: Added issues to create a better location to point people towards
#63 and https://github.com/Transport-for-the-North/.github/issues/31

**Review by BenTaylor-TfN — APPROVED** [review]
